### PR TITLE
Initial version of stats files and path definitions

### PR DESCRIPTION
**Pull request by sdrewc**
Adds the following files:
- routes_stats: statistics on routes for defined time periods
- trips_stats: statistics on trips for defined time periods
- stop_times_stats: statistics on stop_times for defined time periods
- paths: defines paths as groups of routes, or route-segments by defining a first_stop_id and last_stop_id
- paths_stats: statistics on paths for defined time periods

### CODE REVIEW

File: files/paths_stats.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Path Performance Information
```
Comment (e-lo) [inline files/paths_stats.md]: Per our definitions in our glossary for DPA, a path is something a person takes; a route is a vehicle.  In real life though, the difference is ambiguous.  Wondering if we should use a term that makes it obvious that is a collection....like collection, pooled, assembly, medley, group etc.

File: files/paths_stats.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+### A path is a collection of routes that serve a common corridor or origin-destination pair
+
+ *  Filename MUST be `paths.txt`
```
Comment (e-lo) [inline files/paths_stats.md]: This should probably be `***_stats.txt` ?

File: files/paths_stats.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ *  The first line of each file MUST contain case-sensitive field names.
+ *  Field names MUST NOT contain tabs, carriage returns or new lines.
+
+File MUST contain ONE of the following TWO sets of required attributes:
+
+**Option One**:
+
+Required Attributes	| Description										
+----------			| -------------		
+`path_id`			| ID that uniquely identifies a path.
```
Comment (e-lo) [inline files/paths_stats.md]: change to whatever word we end up using.

File: files/paths.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ *  File MUST be a valid CSV file.
+ *  The first line of each file MUST contain case-sensitive field names.
+ *  Field names MUST NOT contain tabs, carriage returns or new lines.
+
+File MUST contain the following required attributes:
+
+Required Attributes	| Description										
+----------			| -------------		
+`path_id`			| ID that uniquely identifies a path.
+`route_id`			| ID that uniquely identifies a route.
```
Comment (e-lo) [inline files/paths.md]: If we want to accumulate among many routes, would we have to denote every route that is in the group? Just trying to think if there is a way to do this automagically (i.e. by selecting from a set of nodes that it must traverse)

**Review by e-lo — CHANGES_REQUESTED** [review]
Please add a README.txt with same bones as GTFS-PLUS

File: files/paths.md (hunk 4 above)
Comment (sdrewc) [reply files/paths.md]: Yes.  If you think about Market Street there are curb-side and island stops with different stop ids but that you might want to include in a single corridor, which would be difficult to do if you used stop sequences.  I think there's probably a trade off between flexibility and automation.  I'm open to reconsidering

File: files/paths_stats.md (hunk 2 above)
Comment (sdrewc) [reply files/paths_stats.md]: oops.

File: files/paths.md (hunk 4 above)
Comment (e-lo) [reply files/paths.md]: what about creating lists where everything across the line is an AND but within the list is an OR?
set_id, from_node_set, to_node_set, thru_node_set
castro_VN, [castro_muni], [van_ness_muni],[]
castro_VN,[castro/17th, 17th/castro etc.],[van_ness/Market, Market/Van Ness, etc], []

File: files/paths_stats.md (hunk 1 above)
Comment (sdrewc) [reply files/paths_stats.md]: how about group or route_group?

Comment (e-lo) [reply files/paths_stats.md]: sounds good

File: files/paths.md (hunk 4 above)
Comment (sdrewc) [reply files/paths.md]: What if you wanted to make sure somethings was NOT included?  For example, maybe you want `Geary_ParkPresidio_Market` but want to exclude the 91?

Comment (sdrewc) [reply files/paths.md]: oh nevermind...

**Comment by sdrewc** [comment]
Made several changes to address comments, including renaming path -> group and revising how groups are defined to use from, to, thru.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+### Changelog
+
+-  `0.1.0`: initial commit
+
+# Specification
+
+GTFS-STAT is an extension to a GTFS transit network with additional files that contain 
+performance data.  
+
+A GTFS-PLUS transit network MAY include the following files:
```
Comment (e-lo) [inline README.md]: PLUS-->STAT

**Review by e-lo — CHANGES_REQUESTED** [review]
one typo

**Review by e-lo — APPROVED** [review]

**Pull request merged**